### PR TITLE
Update FSLogix min version and various links

### PR DESCRIPTION
**Pull request by gacarini**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/sync-vdi-support.md [hunk 1]
```diff
@@ -27,19 +27,19 @@ description: "Learn about the OneDrive sync app support for desktop virtualizati
 For all [supported operating systems](https://support.office.com/article/cc0cb2b8-f446-445c-9b52-d3c2627d681e), the OneDrive sync app supports:
 
 - Virtual desktops that persist between sessions. 
-- Non-persistent virtual desktops that use [Windows Virtual Desktop](https://azure.microsoft.com/services/virtual-desktop/).
-- Non-persistent virtual desktops that have [FSLogix Apps](/fslogix/configure-profile-container-tutorial) or [FSLogix Office Container](/fslogix/configure-office-container-tutorial), and a Microsoft 365 subscription for all of the following operating systems:
+- Non-persistent virtual desktops that use [Azure Virtual Desktop](https://docs.microsoft.com/azure/virtual-desktop/).
```
Comment (JohanFreelancer9) [inline OneDrive/sync-vdi-support.md:30]:
```suggestion
- Non-persistent virtual desktops that use [Azure Virtual Desktop](/azure/virtual-desktop).
```

File: OneDrive/sync-vdi-support.md [hunk 2]
```diff
@@ -27,19 +27,19 @@ description: "Learn about the OneDrive sync app support for desktop virtualizati
...
+- Non-persistent virtual desktops that have [FSLogix Apps](https://docs.microsoft.com/fslogix/configure-profile-container-tutorial) or [FSLogix Office Container](https://docs.microsoft.com/fslogix/configure-office-container-tutorial), and a Microsoft 365 subscription for all of the following operating systems:
```
Comment (JohanFreelancer9) [inline OneDrive/sync-vdi-support.md:31]:
```suggestion
- Non-persistent virtual desktops that have [FSLogix Apps](/fslogix/configure-profile-container-tutorial) or [FSLogix Office Container](/fslogix/configure-office-container-tutorial), and a Microsoft 365 subscription for all of the following operating systems:
```

File: OneDrive/sync-vdi-support.md [hunk 3]
```diff
@@ -27,19 +27,19 @@ description: "Learn about the OneDrive sync app support for desktop virtualizati
...
   - Windows Server 2019 (supports VHDX)
   - Windows Server 2016 (supports VHDX)
   - Windows Server 2012 R2 (supports VHDX)
   - Windows Server 2008 R2 (supports VHD)
-
- Using the OneDrive sync app with non-persistent environments requires that you [install the sync app per machine](./per-machine-installation.md).
  
 > [!NOTE]
-> The minimum supported versions are: OneDrive 19.174.0902.0013 and FSLogix Apps [2.9.7486.53382](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RE4C5aJ).
+> The minimum supported versions are: OneDrive 19.174.0902.0013 and FSLogix Apps [2.9.7653.47581](https://docs.microsoft.com/fslogix/whats-new).
```
Comment (JohanFreelancer9) [inline OneDrive/sync-vdi-support.md:40]:
```suggestion
> The minimum supported versions are: OneDrive 19.174.0902.0013 and FSLogix Apps [2.9.7653.47581](/fslogix/whats-new).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @gacarini, kindly commit my suggested changes to change the links to relative links. Many thanks.

**Comment by gacarini** [comment]
@JohanFreelancer9  - I believe I applied all your suggestions.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@adeejoseph PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**